packages/lodestar/src/chain/forkChoice/statefulDag/lmdGhost.ts

```diff
@@ -197,7 +197,7 @@ export class StatefulDagLMDGHOST implements ILMDGHOST {
   public async start(genesisTime: number): Promise<void> {
     this.genesisTime = genesisTime;
     const numSlot = computeSlotsSinceEpochStart(this.config, getCurrentSlot(this.config, this.genesisTime));
-    const timeToWaitTillNextEpoch = (this.config.params.SLOTS_PER_EPOCH - numSlot) * 
+    const timeToWaitTillNextEpoch = (this.config.params.SLOTS_PER_EPOCH - numSlot) *
       this.config.params.SECONDS_PER_SLOT * 1000;
```

Comment:
This would probably benefit from clock I'm adding in https://github.com/ChainSafe/lodestar/pull/648/files#diff-c823e58d682201704d876febbedd0d52R4

Reply:
In that case, should I wait for that to be merged before merging this?

Reply:
yeah let's use the new clock in @mpetrunic PR, so please hold on a little bit.